.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "sentry-contrib-native-sys/sentry-native"]
 	path = sentry-contrib-native-sys/sentry-native
-	url = https://github.com/getsentry/sentry-native.git
+	url = https://github.com/EmbarkStudios/sentry-native.git
```

Comment:
Not planning on relying on anything else than the official repo.

Reply:
I fixed this in #2